packages/common/src/events/_internal/PropertyMapper.ts

```diff
@@ -109,9 +108,6 @@ export class EventPropertyMapper extends PropertyMapper<
 
     obj.links = computeLinks(store as IEvent);
     obj.slug = getEventSlug(store as IEvent);
-    if (!obj.thumbnailUrl) {
```

Comment:
Looks like `getEventThumbnail` is also called from [computeLinks](https://github.com/Esri/hub.js/blob/12878/new-events-custom-thumbnail/packages/common/src/events/_internal/computeLinks.ts#L22) which is related to populating the event thumbnail on `IHubSearchResult` objects for Events 3 events. Unsure if that is still needed or not. Please verify if Events 3 event search results display as expected after making ^ changes, or if we also need to update the code in `computeLinks` module.